apps/web/src/index.css

```
@@ -1,18 +1,122 @@
+@import url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,300;0,400;0,500;0,700;1,400&family=Instrument+Serif:ital@0;1&family=Sora:wght@300;400;500;600;700&display=swap");
```

Stylelint (17.10.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,300;0,400;0,500;0,700;1,400&family=Instrument+Serif:ital@0;1&family=Sora:wght@300;400;500;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,300;0,400;0,500;0,700;1,400&family=Instrument+Serif:ital@0;1&family=Sora:wght@300;400;500;600;700&display=swap"" (import-notation)

(import-notation)